index.html

```diff
@@ -96,7 +106,17 @@ <h4> Multi-Post<br/>Stories </h4>
           <li><a id="language-name" href="#">css</a></li>
           <li><a id="language-name" href="#">javascript</a></li>
         </ul>
```

Comment:
- [ ] Please make sure to update the main page so that the entire project section in the `index.html` is created dynamically using the information stored in that JavaScript array. All of the HTML in that section should be created when the page loads as requested in the project requirements.